lib/database/user.php

```diff
@@ -29,25 +28,23 @@ function getAccountDetails(&$user, &$dataOut): bool
         return false;
     }
 
-    $dataOut = Cache::store('array')->rememberForever($user . ':account-details', function () use ($user) {
-        sanitize_sql_inputs($user);
-
-        $query = "SELECT ID, User, EmailAddress, Permissions, RAPoints, RASoftcorePoints, TrueRAPoints,
-                         cookie, websitePrefs, UnreadMessageCount, Motto, UserWallActive,
-                         APIKey, ContribCount, ContribYield,
-                         RichPresenceMsg, LastGameID, LastLogin, LastActivityID,
-                         Created, DeleteRequested, Untracked
-                    FROM UserAccounts
-                    WHERE User='$user'
-                    AND Deleted IS NULL";
-
-        $dbResult = s_mysql_query($query);
-        if (!$dbResult || mysqli_num_rows($dbResult) !== 1) {
-            return false;
-        }
+    sanitize_sql_inputs($user);
+
+    $query = "SELECT ID, User, EmailAddress, Permissions, RAPoints, RASoftcorePoints, TrueRAPoints,
+                     cookie, websitePrefs, UnreadMessageCount, Motto, UserWallActive,
+                     APIKey, ContribCount, ContribYield,
+                     RichPresenceMsg, LastGameID, LastLogin, LastActivityID,
+                     Created, DeleteRequested, Untracked
+                FROM UserAccounts
+                WHERE User='$user'
+                AND Deleted IS NULL";
+
+    $dbResult = s_mysql_query($query);
+    if (!$dbResult || mysqli_num_rows($dbResult) !== 1) {
+        return false;
+    }
 
-        return mysqli_fetch_array($dbResult);
-    });
+    $dataOut = mysqli_fetch_array($dbResult);
```

Comment:
this is going to supercede #1151

Reply:
I presumed so; wasn't sure thought. Good to know!